op-supervisor/supervisor/service.go

```diff
@@ -186,11 +186,14 @@ func (su *SupervisorService) initDBSync(ctx context.Context, cfg *config.Config)
 		DataDir: cfg.Datadir,
 		Logger:  su.log,
 	}
-	depSet, err := cfg.DependencySetSource.LoadDependencySet(ctx)
+	// TODO: it seems a bit wasteful to load the full config set just to get the list of chains.
```

Comment:
This might need a follow-up issue. But I don't think it's a big deal, we can also drop the TODO.

Reply:
OK removed the TODO comment